### DISCUSSION TITLE
Edit metadata tab label

### DISCUSSION THREAD

**Discussion by Jonkat323** [open-sdg/open-sdg#1978] [Q&A] [answered]
Hi,

How can I edit a metadata tab label? For example, I want to change "National Metadata" to "COJ Metadata".

Thanks.

**Comment by Jonkat323** [comment]
I am still struggling to figure this out.

**Comment by Govej** [comment, marked as answer]
Hi Jonkat, apologies for missing this query

To edit the metadata tab labels please refer to this part of the read the docs documentation - https://open-sdg.readthedocs.io/en/latest/configuration/#metadata_tabs

It explains how to edit your config file to change this from the default "National Metadata" to your custom name

Any further issues feel free to drop another message, I'll keep an eye on this thread!

James

**Comment by Govej** [comment]
Query answered in issue #1986
